distbench_test_sequencer_test.cc

```diff
@@ -1194,10 +1206,23 @@ tests {
   ASSERT_NE(dest_rpc, dest->second.rpc_logs().end());
   ASSERT_TRUE(dest_rpc->second.failed_rpc_samples().empty());
   ASSERT_EQ(dest_rpc->second.successful_rpc_samples_size(), 25);
+  int64_t iterations = 0;
```

Comment:
Maybe iterations_bits or something like that, to make it easier to understand that it's not just a count.